code-experiments/test/regression-test/test_suites.py

```diff
@@ -67,7 +67,7 @@ def regression_test_a_suite(suite_name, filename):
             try:
                 assert is_equal(f.constraint(x), xfc_dict[key][1])
             except AssertionError:
-                print(f.name, "index,x =", key, "stored x,f,con =",
+                print(f.name, "index,x =", key, "stored f,con =",
```

Comment:
What is the reason for the disappearing `x` here? The `xfc_dict` was not changed in this pull request, or was it?

Reply:
The ```x``` was removed because ```xfc_dict[key]``` contains two values only: ```f``` and ```con```. Therefore, the printed string ```"stored x,f,con ="``` wasn't reflecting the actual printed values (```f```, ```con```). The ```xfc_dict``` was not changed, the ```x``` was a typo.

Reply:
If this is the case, we should rename the `xfc_dict` to `fc_dict`, shouldn't we?

Reply:
The dictionary is indexed by keys of the form ```(index, x)```, maybe that's why there's an ```x``` in the name. Whatever we decide, the renaming is easy to do.